src/helpers/isEnterKeyPress.js

```diff
@@ -0,0 +1,13 @@
+import { codes } from './constants.js'
+
+export const isEnterKeyPress = (event) => {
+  if (
+    event.key === 'Enter' ||
```

Comment:
I'd just return the result of evaluating these like `return foo == bar || baz == qux`(and so on, but it'd be one statement and actually easier to read I think)

Reply:
Noted, will revise.

Reply:
e6c95bd